examples/differential_rotation/differentially_rotated_coordinate.py

```diff
@@ -0,0 +1,59 @@
+"""
+====================================
+Differentially rotating a coordinate
+====================================
+
+How to differentially rotate a coordinate.
+
+The example uses the `~sunpy.coordinates.metaframes.RotatedSunFrame` coordinate
+metaframe in `sunpy.coordinates` to apply differential rotation to a
+coordinate.  See :ref:`sunpy-coordinates-rotatedsunframe` for more details on
+using `~sunpy.coordinates.metaframes.RotatedSunFrame`.
+"""
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sunpy.coordinates import RotatedSunFrame
+from sunpy.data.sample import AIA_171_IMAGE
+from sunpy.map import Map
+
+##############################################################################
+# First, load an AIA observation and define a coordinate in its coordinate
+# frame (here, helioprojective Cartesian).  The appropriate rate of rotation
+# is determined from the heliographic latitude of the coordinate.
+
+m = Map(AIA_171_IMAGE)
```

Comment:
I am not sure we should be using single letter variable names in our example gallery? Maybe `aiamap` is better?

Reply:
Changed